terraform/modules/database/rds.tf

```diff
@@ -2,8 +2,23 @@ resource "aws_db_subnet_group" "cluster" {
   subnet_ids = var.vpc.aws_subnets_private[*].id
 }
 
+resource "aws_kms_key" "aurora" {
+  enable_key_rotation = true
+  tags                = merge(var.tags, { Name = "${var.name}-db-key" })
```

Comment:
Could do with a ProtectsSensitiveData tag too, as per the Tagging policy.

Reply:
Done.